src/app/config/hypeEvmStrategies.ts

```diff
@@ -0,0 +1,82 @@
+import { Strategy } from "../types";
+
+import {
+  POINTS_ID_HYPERBEAT_S1,
+  // POINTS_ID_HYPERLEND_S1,
+  // POINTS_ID_HYPURRFI_S1,
+  // POINTS_ID_HYPERSWAP_S1,
+  // POINTS_ID_TIMESWAP_S1,
+  // POINTS_ID_SILHOUETTE_S1,
+  // POINTS_ID_UPSHIFT_S1,
+  POINTS_ID_SENTIMENT_S1,
+} from "./constants";
+
+export const HYPE_EVM_CONFIG: Strategy[] = [
+  {
+    strategy: "Hyperbeat: hbHype",
+    start: "Jun-06-2025 3:02:03 PM UTC",
+    // owner: "0xd048870caa5a3037f507583b4762a7598251a2fc", // Example whale with points
+    owner: "0x49532bc64C515D6bfe061d44EE0834a996b7c684", // Actual test address
+    fixedValue: { value: 0.2, asset: "HYPE" },
+    points: [
+      {
+        type: POINTS_ID_HYPERBEAT_S1,
+        expectedPointsPerDay: { value: 1, baseAsset: "USD" },
```

Comment:
what're you thinking for this value?

Reply:
Was subconsciously leaving as a problem for future us, but we should probably display "n/a" or "-" if the strategy has no obvious distribution mechanism. I'll add that.

In the future, we can try to calculate the amount a user should get, but this can be tricky. We'd have to get the data (amount distributed that week, and user share) which isn't straight forward, and also try to reverse engineer their distribution mechanism, which we know is not easy and often not accurate.